ui/app/components/clients/attribution.js

```diff
@@ -47,6 +45,11 @@ import { format, isSameMonth } from 'date-fns';
 export default class Attribution extends Component {
   @tracked showCSVDownloadModal = false;
   @service download;
+  attributionLegend = [
```

Comment:
Now that charts will be displaying different data depending on the tab, it made more sense to move the legend definition from the `token.js` parent file to the components rendering the charts

Reply:
For now, no. I think ideally in the future each tab has an attribution chart but that relies on the backend sorting the counts in descending order for each sub-key (i.e `secret_syncs` or `entity_clients`+`non_entiy_clients`)

It's a pretty easy refactor to change how/where this array is passed so I wanted it to be in the easiest to find location for now.